modules/virt-enabling-disabling-vm-delete-protection-web.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the web console
+
+To prevent the inadvertent deletion of a virtual machine, you can enable virtual machine delete protection by using the {product-title} web console.
+
+.Prerequisites
+
+* You have installed {VirtProductName} 4.19 or later.
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
In the case of web console, users do not need `oc`. IMHO, we can drop this prerequisite